### DISCUSSION TITLE
Adding OpenFabric service 'FRRFabric' as part of the default services upon starting core-gui.

### DISCUSSION THREAD

**Discussion by Rinzx** [coreemu/core#738] [Q&A] [answered]
Can you advise how I can tweak the frr.py file to achieve having the FRRFabric service as part of the default services along side ISIS, BGP etc. Thank you.

**Comment by bharnden** [comment, marked as answer]
Assuming you are using the latest release, you can follow this documentation related to adding [new services](https://coreemu.github.io/core/configservices.html#new-services).

Then you could base your Fabric variation on one of the other existing services associated with FRR:
https://github.com/coreemu/core/blob/master/daemon/core/configservices/frrservices/services.py

Replace the config content with what you need for fabric using FRR documentation and choose a new name to give it. You could even place it in the same group.
